src/infra/account/secureAccessKey/secureAccessKeyCmdRepo.go

```diff
@@ -0,0 +1,202 @@
+package secureAccessKeyInfra
+
+import (
+	"errors"
+	"os"
+
+	"github.com/goinfinite/os/src/domain/dto"
+	"github.com/goinfinite/os/src/domain/valueObject"
+	accountInfra "github.com/goinfinite/os/src/infra/account"
+	infraHelper "github.com/goinfinite/os/src/infra/helper"
+	internalDbInfra "github.com/goinfinite/os/src/infra/internalDatabase"
+	dbModel "github.com/goinfinite/os/src/infra/internalDatabase/model"
+)
+
+type SecureAccessKeyCmdRepo struct {
```

Comment:
If you want to keep the keys logic outside of Account repo, that's fine, on the infra layer. It does not need to be an interface on Domain, got it?